app/Filters/AlertsFilter.php

```diff
@@ -26,6 +26,8 @@ public function before(RequestInterface $request, $arguments = null)
      */
     public function after(RequestInterface $request, ResponseInterface $response, $arguments = null)
     {
+        helper('alerts');
```

Comment:
I have already prepared a PR)

Reply:
I would remove phpdoc. It contradicts the interface. This is a question for the framework. Need to either fix `void` to `null` in the interface
```php
    /**
     * @param array|null $arguments
     *
     * @return mixed
     */
```